OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -51,17 +58,22 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager,
 			{
 				// Expand the stats window to cover the hidden objectives
 				var objectiveGroup = widget.Get("OBJECTIVE");
-				var statsHeader = widget.Get("STATS_HEADERS");
 
 				objectiveGroup.Visible = false;
 				statsHeader.Bounds.Y -= objectiveGroup.Bounds.Height;
 				playerPanel.Bounds.Y -= objectiveGroup.Bounds.Height;
 				playerPanel.Bounds.Height += objectiveGroup.Bounds.Height;
 			}
 
+			// orderManager.LocalClient is null when watching a replay
```

Comment:
This comment would live better at the `OrderManager.LocalClient` property itself as a `summary`.

Reply:
added a summary